azkaban-common/src/main/java/azkaban/executor/ExecutionFlowDao.java

```diff
@@ -137,6 +138,30 @@ public void uploadExecutableFlow(final ExecutableFlow flow)
     }
   }
 
+  public List<ExecutableFlow> fetchStaleFlows(final long beforeInMs)
```

Comment:
beforeInMs is not clear, is it microseconds or milliseconds?
Please consider using full name.

Reply:
Will do.